compiler/src/ammtoaga/index.ts

```diff
@@ -286,7 +286,7 @@ const loadStatements = (
       // It's a closure, skip it
       continue
     }
-    const node = depGraph.byLP.get(statement)
+    const node = depGraph.byLP.get(statement) || null
```

Comment:
Do you still need this `|| null`? If so, why?

Reply:
yes, the `Map.get` call can still returned `undefined`

Reply:
But this is the statement-based querying. It *should* have already seen all potential dependencies in the `Map` and never return `undefined`. That implies a bug somewhere else is being covered up here.